sdk/communication/communication-phone-numbers/test/lro.purchase.spec.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure/test-utils-recorder";
+import { assert } from "chai";
+import {
+  PhoneNumberSearchRequest,
+  KnownPhoneNumberType,
+  KnownPhoneNumberAssignmentType,
+  PhoneNumberSearchResult
+} from "../src";
+import { PhoneNumbersClient } from "../src/phoneNumbersClient";
+import { createRecordedClient, testPollerOptions } from "./utils/recordedClient";
+
+describe("PhoneNumbersClient - lro - purchase", function() {
+  let recorder: Recorder;
+  let client: PhoneNumbersClient;
+  let includePhoneNumberLiveTests: boolean;
+  const countryCode = "US";
+
+  beforeEach(function() {
+    ({ client, recorder, includePhoneNumberLiveTests } = createRecordedClient(this));
+  });
+
+  afterEach(async function() {
+    if (!this.currentTest?.isPending()) {
+      await recorder.stop();
+    }
+  });
+
+  describe("successfully purchases a phone number", function() {
+    let searchResults: PhoneNumberSearchResult;
+
+    it("finds phone number to purchase", async function() {
+      if (!includePhoneNumberLiveTests) {
+        this.skip();
+      }
+
+      const searchRequest: PhoneNumberSearchRequest = {
+        phoneNumberType: KnownPhoneNumberType.TollFree,
+        assignmentType: KnownPhoneNumberAssignmentType.Application,
+        capabilities: {
+          sms: "inbound+outbound",
+          calling: "none"
+        },
+        areaCode: "833",
```

Comment:
We had out of stock issues, and Adrian has implemented optional areaCodes for toll-free numbers. Hopefully we can remove areaCode here and make the request then more likely to succeed.